packages/fast-components-styles-msft/src/slider/index.ts

```diff
@@ -70,18 +70,33 @@ const styles: ComponentStyles<SliderClassNameContract, DesignSystem> = {
         "&:active": {
             background: neutralForegroundActive,
         },
+        "@media (-ms-high-contrast:active)": {
+            background: "ButtonText",
+            "&:hover": {
+                background: "Highlight",
```

Comment:
Seems like this is CSS2 keywords https://www.w3.org/TR/css-color-3/#css2-system

CSS2 keywords are marked as deprecated, why are we using them here?

Reply:
This is - AFAIK - the only way to tap into the Windows high contrast theme. This is what we do for all other components to support high contrast.

Reply:
Yeah, I think this is currently the design/mapping. This article by Greg Whitworth (MSFT Edge) lays it out in more detail: http://www.gwhitworth.com/blog/2017/04/how-to-use-ms-high-contrast/

Reply:
These keywords have been added as part of a new list of system colors.
https://github.com/w3c/csswg-drafts/issues/3804

Reply:
Thanks, just wanted to get this on the record